### DISCUSSION TITLE
Mysterious Devices Appearing in Device List

### DISCUSSION THREAD

**Discussion by macelinq** [rustdesk/rustdesk-server-pro#403] [Q&A]
We just signed up for a 30 day trial of RustDesk and set up our server. We have added a few devices for testing purposes. We had a device we did not recognize appear in our device list from an IP address well outside our IP ranges. I disabled the first device and immediately another device appeared. I deleted both devices and not long after a couple more devices appeared that were not ours. 
Any ideas where these devices are coming from? 
Is this common? 

![Screenshot 2024-10-03 140902](https://github.com/user-attachments/assets/7309003d-6fcf-4b8c-9e5a-1c355de31938)

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk-server-pro/discussions/307
